src/platform/packages/private/kbn-esql-editor/src/esql_editor.tsx

```diff
@@ -849,8 +850,14 @@ export const ESQLEditor = memo(function ESQLEditor({
                       monaco.KeyMod.CtrlCmd | monaco.KeyCode.Slash,
                       onCommentLine
                     );
-
                     setMeasuredEditorWidth(editor.getLayoutInfo().width);
+                    if (expandToFitQueryOnMount) {
+                      const lineHeight = editor.getOption(monaco.editor.EditorOption.lineHeight);
+                      const lineCount = editor.getModel()?.getLineCount() || 1;
+                      const padding = lineHeight * 1.25; // Extra line at the bottom, plus a bit more to compensate for hidden vertical scrollbars
```

Comment:
Do we need this? From my tests it works ok without it. Can you share with me an example that it doesnt? I am trying to avoid random constants unless there is a good reason to have them

Reply:
https://github.com/user-attachments/assets/14cb4427-a8bf-4d70-8a0d-2e92f711d965

Tested on Firefox Mac, here's what happens when I remove the extra .25 of padding at the bottom. Note the vertical scrollbar for the flyout appears on the far right hand side of the screen when I click the Test Query button, which narrows the element width by just enough pixels to mess with the line length.